### PR TITLE
(doc) Added some changes based on running through this myself

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -31,16 +31,19 @@ You can also install Vagrant/Virtualbox on Windows by running `choco install pac
...
- * Now the box is ready for you to start testing against. Run `vagrant sandbox on`.
+ * Run `vagrant up` to prepare the machine for testing. 
+   * **Note** due to the way that vagrant works, the first time that you run this command, the vagrant box named __ferventcoder/win2012r2-x64-nocm__ needs to be downloaded from the [Atlas website](https://atlas.hashicorp.com/ferventcoder/boxes/win2012r2-x64-nocm).  This will take quite a while, and should only be attempted on a reasonably fast connection, that doesn't have any download limit restrictions. Once it has downloaded it will import the box and apply the scripts and configurations to the box as listed inside the `Vagrantfile`.  You can find the downloaded box in the `~/.vagrant.d` or `c:\users\username\.vagrant.d` folder.
+ * Now the box is ready for you to start testing against. 
+ * Using the [vagrant plugin](https://github.com/jedi4ever/sahara) that was installed earlier, it is now possible to take a snapshot of the VM, so that after testing packages, the VM can be returned to this known "good" state.  This snapshot can be taken using the following command: `vagrant sandbox on`.
```
Comment (ferventcoder) [inline ReadMe.md]: This is informational. However there still needs to be an imperative. 

Run `vagrant sandbox on`.

**Comment by ferventcoder** [comment]
Other than ensuring that we are directing the user to run a command, :+1:

File: ReadMe.md (hunk 1 above)
Comment (gep13) [inline ReadMe.md]: As in move the command to the start of the bullet point, rather than having it at the end?

Comment (ferventcoder) [inline ReadMe.md]: Yeah - "Run `vagrant sandbox on`. Using the..."

**Comment by gep13** [comment]
@ferventcoder fixed up.

**Comment by ferventcoder** [comment]
Thanks!

**Pull request merged**